metadata-ingestion/src/datahub/ingestion/source/powerbi/rest_api_wrapper/data_resolver.py

```diff
@@ -198,6 +198,7 @@ def get_dashboards(self, workspace: Workspace) -> List[Dashboard]:
                 webUrl=instance.get(Constant.WEB_URL),
                 workspace_id=workspace.id,
                 workspace_name=workspace.name,
+                usageMetrics={},
```

Comment:
what's the difference between None and {} here? does it make sense to allow both?